src/instances/RaidInstance.ts

```diff
@@ -241,6 +246,21 @@ export class RaidInstance {
     private static readonly DEFAULT_EMBED_COLOR: number = 16777215; //default to white
     private _embedColor: number;
 
+    // The raid instance start time and expiration time
+    private _startTime: number;
+    private _expTime: number;
+    private static readonly DEFAULT_RAID_DURATION: number = 60*60*1000; //1 hour in milliseconds
+
+    // Instance information for logging
+    private readonly _instanceInfo: string;
+
+    // Time between panel updates in ms
+    private readonly _intervalDelay: number = 5000;
+
+    // Temporary Alert Duration
+    private readonly _tempAlertDelay: number = 10*60*1000; //Ten minutes
```

Comment:
If this isn't going to change between instances, make this `private static readonly ...`